### PR TITLE
Adding language files (Polish) and repair editing own avatar function

### PR DESCRIPTION
**Pull request by Monter**

### CODE REVIEW

**Comment by ronalfy** [comment]
@Monter is the sole purpose of the plugin changes to add the title attribute to the image? Just clarifying.

**Comment by Monter** [comment]
No. Please start the plug-in, add your own avatar and see what happens when you want to click on it to change. Link not only does not have a title, but also directs the wrong place (yourblog.domain/?mt_pp= [id]). This is caused by an erroneous construction of the link to the picture, my amendment improves it.

File: metronet-profile-picture.php [hunk 1]
```diff
@@ -109,7 +109,8 @@ public function ajax_get_thumbnail() {
 		$post_id = isset( $_POST[ 'post_id' ] ) ? absint( $_POST[ 'post_id' ] ) : 0;
 
 		if ( has_post_thumbnail( $post_id ) ) {
-			$post_thumbnail = get_the_post_thumbnail( $post_id, 'thumbnail' );
+			$thumb_src = wp_get_attachment_image_src( get_post_thumbnail_id($post_id), 'thumbnail' , false, '' );
```
Comment (ronalfy) [inline metronet-profile-picture.php]: Arguments need to be separated with spaces. Ex: `get_post_thumbnail_id( $post_id )`

File: metronet-profile-picture.php [hunk 2]
```diff
@@ -109,7 +109,8 @@ public function ajax_get_thumbnail() {
...
+			$post_thumbnail = sprintf( '<img src="%s" width="150" height="150" title="%s" />', $thumb_src[0], esc_attr__( "Upload or Change Profile Picture", 'metronet-profile-picture' ) );
```
Comment (ronalfy) [inline metronet-profile-picture.php]: $thumb_src[0] needs to be escaped using `esc_url`

File: metronet-profile-picture.php [hunk 3]
```diff
@@ -367,7 +368,8 @@ public function insert_upload_form() {
 					if ( has_post_thumbnail( $post_id ) ) {
 						$has_profile_image = true;
 						echo '<a href="#" class="mpp_add_media">';
-						$post_thumbnail = get_the_post_thumbnail( $post_id, 'thumbnail' );
+						$thumb_src = wp_get_attachment_image_src( get_post_thumbnail_id($post_id), 'thumbnail' , false, '' );
```
Comment (ronalfy) [inline metronet-profile-picture.php]: Again with spaces for inbetween arguments.

File: metronet-profile-picture.php [hunk 4]
```diff
@@ -367,7 +368,8 @@ public function insert_upload_form() {
...
+						$post_thumbnail = sprintf( '<img src="%s" width="150" height="150" title="%s" />', $thumb_src[0], esc_attr__( "Upload or Change Profile Picture", 'metronet-profile-picture' ) );
```
Comment (ronalfy) [inline metronet-profile-picture.php]: Escape $thumb_src[0]

**Comment by ronalfy** [comment]
@Monter thanks, that makes sense. Could you paste the HTML you're getting? I'm not seeing the link you are mentioning. 

Could something be inadvertently hooking into `get_the_post_thumbnail` and adding the extra HTML you mentioned?

**Comment by Monter** [comment]
Fixed.

**Comment by Monter** [comment]
This is HTML code after adding my own picture (before changes in this pull request):
wrong_link
After changes everything work O.K.

**Comment by ronalfy** [comment]
Ah, yeah, something is definitely hooking into that filter and inserting a link around the image. I'll push out an update tonight if I have time. Thanks for the PR.

**Pull request merged**